test/e2e/framework/auth/helpers.go

```diff
@@ -92,23 +106,30 @@ func BindClusterRole(c v1beta1rbac.ClusterRoleBindingsGetter, clusterRole, ns st
 		Subjects: subjects,
 	})
 
-	// if we failed, don't fail the entire test because it may still work. RBAC may simply be disabled.
 	if err != nil {
-		klog.Errorf("Error binding clusterrole/%s for %q for %v\n", clusterRole, ns, subjects)
+		return fmt.Errorf("error binding clusterrole/%s for %q for %v: %v", clusterRole, ns, subjects, err)
```

Comment:
Added the err to the message here; not sure why it wasn't included in the first place.